Create ui-c8y-1018-503-53-configuration-ui-gets-supported-security-mode-list-from-the-lwm-2.md (#1766)
SoftwareAG/c8y-docs: content/change-logs/ui-c8y-1018-503-53-configuration-ui-gets-supported-security-mode-list-from-the-lwm-2.md

---
date: ""

> title: configuration UI gets supported security mode list from the lwm2m agent  [GRAFT][release/y2024] (#5659)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Configuration UI Gets Supported Security Mode List from LWM2M Agent
```

product_area: Device management & connectivity
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-3161
version: 1018.503.53
---

> configuration UI gets supported security mode list from the lwm2m agent  [GRAFT][release/y2024] (#5659)

Author: The Claude AI suggested to use the following body:
```suggestion
The Configuration UI now gets the supported security mode list from the LWM2M agent. This allows the UI to display the available security modes for the user to select when configuring the LWM2M client. The supported security modes are retrieved directly from the LWM2M agent via its API.
```

> The supported security modes, which are available for selection on the configuration page, are now retrieved dynamically from the LWM2M agent rather than being hardcoded in the UI.

Reviewer: ```suggestion
The supported security modes which are available for selection on the configuration page are now retrieved dynamically from the LWM2M agent rather than being hardcoded in the UI.
```
